.github/workflows/_rust.yml

```diff
@@ -125,10 +125,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        include:
-          # TODO: Add Windows as part of issue #3782
-          - runs-on: ubuntu-20.04
-          - runs-on: ubuntu-22.04
+        # TODO: Add Windows as part of issue #3782
+        runs-on: [ubuntu-20.04, ubuntu-22.04]
+        test: [linux-group]
```

Comment:
I think we should make it part of the job name too!

Reply:
oops yes